posts/inside-rust/2023-04-12-trademark-policy-draft-feedback.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "Seeking Feedback on a Trademark Policy Draft"
+author: Ryan Levick, Jane Losare-Lusby, Tyler Mandry, Mark Rousskov, Josh Stone, and Josh Triplett
+team: The Foundation Project Directors and informal Trademark Working Group Members
+---
+
+# Seeking Feedback on a Trademark Policy Draft
```

Comment:
```suggestion
# A note on the Trademark Policy Draft
```

I meant to update this since we've already announced we're seeking feedback; this is more about the background and addressing the abuse directed at Foundation staff.